ajax/libs/jquery.x18n/package.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "jquery.x18n",
+  "filename": "jquery.x18n.min.js",
+  "version": "1.1.3",
+  "description": "The jQuery adapter for the x18n internationalisiation library.",
+  "author": {
+    "name": "Florian H.",
+    "url": "https://github.com/js-coder"
```

Comment:
Though author is not an important info for cdnjs now, I believe this should be https://github.com/florian